pvlib/modelchain.py

```diff
@@ -1649,7 +1660,7 @@ def run_model_from_poa(self, data):
         pvlib.modelchain.ModelChain.run_model
         pvlib.modelchain.ModelChain.run_model_from_effective_irradiance
         """
-
+        data = _to_tuple(data)
```

Comment:
Will this alter the user-supplied `data`?

Reply:
No, it will create a new container that refers to the original members of `data`, but they will remain unchanged as long as we don't do anything to change them. Since we copy the data we need internally we should be fine.

```python
xs = [[1, 2], [3, 4]]
ys = tuple(xs)
print(ys)
# ([1, 2], [3, 4])
ys[0].pop()
print(ys)
# ([1], [3, 4])
print(xs)
# [[1], [3, 4]]
```